README.md

```diff
@@ -27,20 +27,19 @@ build collaborative or distributed applications ping us at
 
 ## Sponsors
 
-I'm currently looking for sponsors that allow me to be less dependent on contracting work. These awesome backers already fund further development of Yjs:
+I'm currently looking for sponsors that allow me to be less dependent on
+contracting work.These awesome backers already fund further development of Yjs:
 
 [![Vincent Waller](https://github.com/vwall.png?size=60)](https://github.com/vwall)
-[<img src="https://user-images.githubusercontent.com/5553757/83337333-a7bcb380-a2ba-11ea-837b-e404eb35d318.png" height="60px" />](https://input.com/)
```

Comment:
There is a new  rule for markdown-lint line's max length is less than 80
because I could't find for line break in src url.  I use git.io for that url 
Can I change markdown-lint max length rule 80 to 130?

Reply:
OK, I got it.  I'll rollback that line and add  markdown lint ignore to check line length that line